packages/form-js-viewer/assets/form-js.css

```
@@ -337,3 +428,49 @@
   padding: 6px 8px;
   color: var(--color-text-lighter);
 }
+
+
+
+/**
+* Flatpickr style adjustments
+*/
+
+.flatpickr-day.today {
+  border-color: transparent !important;
+  background-color: transparent !important;
+  font-weight: bold !important;
+}
+
+.flatpickr-day.today:hover,
+.flatpickr-day.today:focus {
+  border-color: var(--color-grey-225-10-55) !important;
+  background-color: var(--color-grey-225-10-55) !important;
```

```suggestion
  background-color: var(--color-background-datetime) !important;
```

Maybe introduce that one to be more specific.